docs/book/src/clusterctl/developers.md

```diff
@@ -27,7 +27,7 @@ Next, create a `clusterctl-settings.json` file and place it in your local copy o
 
 ```yaml
 {
-  "providers": [ "cluster-api", "kubeadm-bootstrap", "aws"],
+  "providers": [ "cluster-api", "kubeadm-bootstrap","kubeadm-control-plane", "aws"],
```

Comment:
The core provider is now called "core" instead of "cluster-api" and is included automatically, so no need to list it here.

```suggestion
  "providers": ["kubeadm-bootstrap","kubeadm-control-plane", "aws"],
```

Reply:
thanks @detiber , good to know!  updated as you suggested.